src/main/java/core/mate/academy/service/ExcavatorProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Excavator;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ExcavatorProducer implements MachineProducer<Excavator> {
+    @Override
+    public List<Excavator> get() {
```

Comment:
Don't begin method implementation with an empty line. Please remove the empty line before the method declaration.